src/josegonzalez/Queuesadilla/Engine/PdoEngine.php

```diff
@@ -229,22 +229,17 @@ public function quoteIdentifier($identifier)
             return $this->startQuote . $identifier . $this->endQuote;
         }
 
+        // string.string
```

Comment:
Why not instead pull the regex out into a variable? That should help explain what the regex is for.